portfolio/src/main/webapp/Image_Gallery.html

```diff
@@ -6,18 +6,43 @@
     <title>Charlie's Gallery</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
```

Comment:
Can you put this style into a CSS file and reference it here?
(Maybe also put a comment there describing what that color is)